cmd/git-chglog/main.go

```diff
@@ -120,6 +132,24 @@ func CreateApp(actionFunc cli.ActionFunc) *cli.App {
 			Usage:  "Regular expression of tag filter. Is specified, only matched tags will be picked",
 		},
 
+		cli.StringFlag{
+			Name:   "jira-url",
+			Usage:  "Jira URL",
+			EnvVar: "JIRA_URL",
+		},
+
+		cli.StringFlag{
+			Name:   "jira-username",
+			Usage:  "Jira username",
+			EnvVar: "JIRA_USERNAME",
+		},
+
+		cli.StringFlag{
+			Name:   "jira-token",
+			Usage:  "Jira token",
+			EnvVar: "JIRA_TOKEN",
+		},
+
```

Comment:
This will be needed after the rebase with current `master` and the bump to `urfave/cli/v2`

```suggestion
		&cli.StringFlag{
			Name:   "jira-url",
			Usage:  "Jira URL",
			EnvVar: "JIRA_URL",
		},

		&cli.StringFlag{
			Name:   "jira-username",
			Usage:  "Jira username",
			EnvVar: "JIRA_USERNAME",
		},

		&cli.StringFlag{
			Name:   "jira-token",
			Usage:  "Jira token",
			EnvVar: "JIRA_TOKEN",
		},

```